src/ristretto.rs

```diff
@@ -167,7 +167,7 @@ use core::borrow::Borrow;
 use rand::{Rng, CryptoRng};
 
 use digest::Digest;
-use generic_array::typenum::U64;
+use digest::generic_array::typenum::U64;
```

Comment:
Might as well use `generic_array` from `digest` rather that requiring a version-lockstepped `extern crate`, eh?